src/main/java/org/fundacionjala/Shapes/Square.java

```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.Shapes;
+
+/**
+ * This class is in charge to calculate the area and perimeter for a Square
+ */
+public class Square implements Shapes {
```

Comment:
In fact Square is a Rectangle with the equals sides.
Change the logic for this class